### PR TITLE
Malware Analysis Notebook v0.3: The Jeffsum Update

### PR DESCRIPTION
**Pull request by HuskyHacks**
Small tweaks to UI based on stream feedback, removing unneeded import, adding instructions to the front page readme for install. And Jeffsum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,34 @@
 
 This repo contains Jupyter Notebooks useful for several aspects of Blue Team work.
 
+## Installation
+
+### Linux
+You need Jupyter Notebooks! On linux, run:
+```
+$ sudo apt-get install jupyter
```
Comment (mttaggart) [inline README.md:22]: Jupyter can be installed with `pip`, so probably should use the same method for everthing.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,34 @@
...
+```
+Clone and enter the directory:
+```
+$ git clone https://github.com/mttaggart/blue-jupyter.git && cd blue-jupyter
+```
+If you don't have it already, install `poetry`:
```
Comment (mttaggart) [inline README.md:28]: Once we have this thing in a stable state, the plan is to also create a proper `requirements.txt`

Comment (HuskyHacks) [reply README.md:28]: Apt-get install replaced with pip

File: README.md (hunk 1 above)
Comment (HuskyHacks) [reply README.md]: Apt-get install replaced with pip

**Pull request merged**